pkg/render/apiserver.go

```diff
@@ -33,6 +32,8 @@ import (
 	apiregv1 "k8s.io/kube-aggregator/pkg/apis/apiregistration/v1"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
+	"github.com/tigera/operator/pkg/render/common/authentication"
+
```

Comment:
Please remove this newline so this import can be groups with the other tigera/operator imports.